common/src/main/java/com/revenuecat/purchases/common/ReplaceSkuInfo.kt

```diff
@@ -4,6 +4,7 @@ import com.android.billingclient.api.BillingFlowParams
 import com.revenuecat.purchases.models.StoreTransaction
 
 data class ReplaceSkuInfo(
-    val oldPurchase: StoreTransaction,
+    val oldPurchase: StoreTransaction, // TODO should we just pass the token?
```

Comment:
we only use the token, so thought we could just pass that instead of the whole transaction..

Reply:
Would a token ever have more than one `StoreTransaction` or product thingy in it? 🤔 Or is an exact 1 to 1 of `StoreTransaction`?

Reply:
great question.... we make `StoreTransaction` out of BillingClient's `Purchase` (docs [here](https://developer.android.com/reference/com/android/billingclient/api/Purchase)), which only has one _token_ per purchase. 

when/if multi-line is released, there could be more than one _product_ returned in the purchase, though..

it is a little confusing that they say in the docs 
```a token that uniquely identifies a purchase for a given item and user pair.``` multi-line sounds like multi-item to me... but they'd need an API change to return multiple tokens in that case, so i think we can safely assume a 1<>1 between token and transaction